pro/src/pages/AdageIframe/app/components/OffersFavorites/__specs__/OffersFavorites.spec.tsx

```diff
@@ -98,7 +99,7 @@ describe('OffersFavorites', () => {
     renderAdageFavoritesOffers(user)
 
     await waitFor(() =>
-      expect(screen.queryByText(/Chargement en cours/)).not.toBeInTheDocument()
+      expect(screen.queryAllByTestId('spinner')).toHaveLength(0)
```

Comment:
Ca me semble bien comme remplacement de query. Dans les skeletons on n'a plus "Chargement en cours", donc ça doit passer de temps en temps quand le chargement est super court c'est ça ?

Par contre pourquoi pas rester sur :
`expect(screen.queryByTestId('spinner')).not.toBeInTheDocument()` ?

Reply:
une préférence en soit il n'y a pas vraiment de raison